index.html

```diff
@@ -50,19 +95,15 @@
   <div class="sample">
     <div id="demo-container" class="demo">
       <example-snippet stamp-to="demo-container" highlight-as="html">
-          <template>
-              <model-viewer
-                src="examples/assets/Astronaut.glb"
-                alt="A 3D model of an astronaut"
-                controls
-                auto-rotate
-                background-color="#455A64"></model-viewer>
-          </template>
-
-          <script>
-  console.log('This is how you log things to the console!');
-</script>
-
+        <template>
+            <model-viewer
+              src="examples/assets/Astronaut.glb"
+              alt="A 3D model of an astronaut"
+              controls
+              auto-rotate
+              background-color="#455A64"></model-viewer>
+        </template>
+        <i></i>
```

Comment:
Does the `<i></i>` do something here?

Reply:
Yeah, it is the node that displays the throbbing arrow icon on the splash page for mobile views